frontend/src/Components/Results/Results.tsx

```diff
@@ -65,6 +112,14 @@ function ResultDisplay(props: ResultDisplayProps) {
           />
         </div>
       </div>
+      <div className="row">
+        <div className="col-lg-6">
+          <BarChart yAxis={cumulativeDeaths} xAxis={agentTypes} graphTitle="Cumulative Deaths per Agent type" />
```

Comment:
I think the title should just be "Deaths per agent type" since the cumulative deaths have been grouped?
Cumulative deaths can be used independently of agent type to show how deaths count evolve over time

Reply:
Done in the most recent commit!